### PR TITLE
eos: Add new states to eos_lacp_interfaces

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY

Add new states gathered , rendered and parsed to eos_lacp_interfaces

related to #23

### CODE REVIEW

File: plugins/modules/eos_lacp_interfaces.py [hunk 1]
```diff
@@ -37,8 +37,7 @@
 }
 
 DOCUMENTATION = """module: eos_lacp_interfaces
-short_description: Manage Link Aggregation Control Protocol (LACP) attributes of interfaces
-  on Arista EOS devices.
+short_description: Lacp_interfaces resource module.
```
Comment (rohitthakur2590) [inline plugins/modules/eos_lacp_interfaces.py:40]: the way I have noticed in other platform modules it should be like "LACP interfaces resource module"

File: plugins/modules/eos_lacp_interfaces.py [hunk 2]
```diff
@@ -69,6 +68,16 @@
...
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the NX-OS device by executing
+        the command B(show running-config | section ^interfaces).
+      - The state I(parsed) reads the configuration from C(running_config) option and transforms
+        it into Ansible structured data as per the resource module's argspec and the value is then
+        returned in the I(parsed) key within the result.
+    type: str
+    version_added: "1.0.0"
```
Comment (rohitthakur2590) [inline plugins/modules/eos_lacp_interfaces.py:80]: need to be on the module instead of options.

File: plugins/modules/eos_lacp_interfaces.py [hunk 3]
```diff
@@ -208,6 +220,61 @@
...
+
+# Using gathered:
+# native config:
+#  veos#show run | section ^interface
+# interface Ethernet1
+#    lacp port-priority 30
+# interface Ethernet2
+#    lacp rate fast
+
+- name: Gather lacp facts from the device
```
Comment (rohitthakur2590) [inline plugins/modules/eos_lacp_interfaces.py:269]: Minor cosmetic suggestion: Gather LACP interfaces facts from the device.

**Comment by GomathiselviS** [comment]
gate

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**